code/modules/power/apc.dm

```diff
@@ -597,9 +597,7 @@
 	interact(user)
 
 /obj/machinery/power/apc/attack_alien(mob/living/carbon/alien/humanoid/user)
-	if(!user)
-		return
-	user.changeNext_move(CLICK_CD_MELEE)
```

Comment:
This is somehow causing a conflict with wires all cut proc on line 605 i think by some reason maybe the wires getting cut proc line 613 with datum cannot properly activate due to the overloading to the datum due to datums being slower in communication to finalize its completion somehow.
This alien attack proc is rather slow by how i am seeing it since it still uses update icon (which is slower than overlay update right?) and could be slowed down somewhat by visible message.
It could of been within the else if check already trying to process it when click restarts the proc possibly.
Best thing to do would be converge the visible messages and update icons to activate below the if else if checks to see if that fixes it. (Aka have three string vars hold the info to transfer to one visible message at end and same with update icon if needed.)